src/JHipsterNetSampleApplication/Data/Extensions/DbContextExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.EntityFrameworkCore;
+
+namespace JHipsterNetSampleApplication.Data.Extensions {
+    public static class DbContextExtensions {
+        public static void AddGraph(this DbContext context, object rootObject)
```

Comment:
For extension method please use `@this` as delegate argument name (rather than context).
See https://github.com/jhipster/jhipster-net-sample-app-template/blob/master/src/JHipsterNetSampleApplication/Infrastructure/MvcStartup.cs#L11